config.tfvars

```diff
@@ -53,6 +53,11 @@ jira_helm_chart_version = "1.2.0"
 # installed and configured.
 jira_replica_count = 1
 
+# Installation timeout
+# Different variables can influence how long it takes the application from installation to ready state. These
+# can be dataset restoration, resource requirements, number of replicas and others.
```

Comment:
It has an impact. While the datasets are restored from the snapshots, the application still needs to do some post-processing when it initialises. For example in the case of Bitbucket, if there are some discrepancies between DB and shared home, it tries to reconcile them so it will eventually start successfully but it takes time to update the paths and fix the repos (in my case it was almost an hour).

Reply:
Dataset restoration affects Jira as well. If the size of the indexSnapshot is big, Jira node needs more time to spin up.

Reply:
I see